startup/uinput.conf

```diff
@@ -0,0 +1,22 @@
+# If using systemd and you want the uinput module to be loaded at boot you can
+# put this file into /etc/modules-load.d/uinput.conf
+# Troubleshooting
+# Run:
+
+# systemctl status systemd-modules-load.service
+
+# The last line contains the PID and exit code. status=0/SUCCESS means the
+# module was successfully inserted and can be confirmed by running:
+
+# journalctl -b _PID=<PID>
+
+# When the exit code is 0/SUCCESS it means the module has been successfully
+# inserted. This can be confirmed by running:
+
+# lsmod | grep uinput
+
+# Credit and a more thorough explanation for auto loading a module checkout
+# this stack exchange thread:
+# https://unix.stackexchange.com/questions/71064/systemd-automate-modprobe-command-at-boot-time
+
+uinput
```

Comment:
> Add uinput.conf to startup files. For systemd users uinput.conf can be used to auto load the uinput module. There is already a file to help for creating a systemd service for Kmonad but without auto loading that module this kind of defeats the purpose of having kmonad start at boot. The file tells a user where to place it and how to verify it is working.

This sounds like weird behaviour to me—I don't think having to manually start `uinput` is very common (at least I have never heard users complain about this, which I'm sure would have happened)